build_tools/archs/Makefile.llvm

```diff
@@ -0,0 +1,36 @@
+CC = mpicc
+CFLAGS = -O3
+
+FC = mpif90
+FFLAGS = -cpp -std=f2018 -fimplicit-none
+# HLFIR is required at least since 7411a59001ca7c77ca374cba2f986dfe614c0462
```

Comment:
I'm not very familiar with the development of Flang - is the commit a particular flang or LLVM commit? And is HLFIR an extension?

Could you perhaps add a (brief) subsection in the build system (perhaps under where you've noted the architecture-specific flags) about the verion(s) of flang that can build CCS?

Reply:
Oh I should have been explicit, it's actually a [commit from CCS](https://github.com/asimovpp/asimov-ccs/commit/7411a59001ca7c77ca374cba2f986dfe614c0462).

HLFIR (High-Level Fortran IR) is another way of lowering Fortran code (it sits on top of FIR—Fortran IR). It's meant to be made the default option at some point, but for now the default is to lower Fortran to FIR directly.

I took note of the commit that made me enable HLFIR lowering as this caused a performance regression that might be worth looking into later.

> Could you perhaps add a (brief) subsection in the build system (perhaps under where you've noted the architecture-specific flags) about the verion(s) of flang that can build CCS?

Yep, will do. Not sure if any tagged release can build CCS currently, but I'll check!